Update tyk-stack-chart.md (#4493)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-charts/tyk-control-plane-chart.md

      value: debug
```

> Example of setting extra [environment variable using ConfigMap data](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#define-container-environment-variables-using-configmap-data), using gateway as an example:

Reviewer: Added suggestion to remove duplication of example, applicable to other chart content (tyk-control-plane-chart, tyk-data-plane-chart, tyk-oss-chart, tyk-stack-chart) files

```suggestion
An example is listed below for setting extra [environment variable using ConfigMap data](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#define-container-environment-variables-using-configmap-data), using gateway:
```

```yaml
tyk-gateway:
  gateway:
    extraEnvs:
    - name: CONFIG_USERNAME
      valueFrom:
        configMapKeyRef: 
          name: backend-user
          key: backend-username
```

> Example of setting extra [environment variable using Secret data](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#define-container-environment-variables-using-secret-data), using gateway as an example:

Reviewer: ```suggestion
An example is listed below for setting extra [environment variable using secret data](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#define-container-environment-variables-using-secret-data), using gateway:
```

```yaml
tyk-gateway:
  gateway:
    extraEnvs:
    - name: SECRET_USERNAME
      valueFrom:
        secretKeyRef: 
          name: backend-user
          key: backend-username
```

> In the above example, an extra environment variable `SECRET_USERNAME` will be added to gateway container, taking value from `backend-username` value defined in the Secret `backend-user`. It is useful if you want to [access secret data from Tyk Gateway configuration file (tyk.conf) or API definitions]({{<ref "tyk-configuration-reference/kv-store#how-to-access-the-externally-stored-data">}}).

Reviewer: ```suggestion
In the above example, an extra environment variable `SECRET_USERNAME` will be added to the Gateway container, with a value of `backend-username` associated with the secret `backend-user`. It is useful if you want to access secret data from [Tyk Gateway configuration file (tyk.conf) or API definitions]({{<ref "tyk-configuration-reference/kv-store#how-to-access-the-externally-stored-data">}}).
```
